developer_setup.md

```diff
@@ -88,6 +88,26 @@
   * [rbenv](https://github.com/sstephenson/rbenv)
 * Required Ruby version is 2.2.2+
 
+#### Example using rbenv
+
+First get the requirements to perform a build of a newer Ruby version
+
+```bash
+sudo yum install -y git-core zlib zlib-devel gcc-c++ patch readline readline-devel libyaml-devel libffi-devel openssl-devel make bzip2 autoconf automake libtool bison curl sqlite-devel
```

Comment:
I think that packages belong up in the OS specific section above with an appropriate comment